api/ratelimit/config/ratelimit/v3/rls_conf.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package ratelimit.config.ratelimit.v3;
+
+option java_package = "io.envoyproxy.ratelimit.config.ratelimit.v3";
+option java_outer_classname = "RlsConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/ratelimit/config/ratelimit/v3;ratelimitv3";
+
+// [#protodoc-title: Rate Limit Config Discovery Service (RLS Conf DS)]
```

Comment:
Remove and replace with more comments.

Reply:
Fixed with https://github.com/envoyproxy/ratelimit/pull/371/commits/0f1a7494234a24262e4f0c5bc00d614f934a8ae3